packages/web/src/layouts/swap/components/swap-card/swap-card-content/SwapCardContent.tsx

```diff
@@ -121,6 +121,10 @@ const SwapCardContent: React.FC<ContentProps> = ({
     return swapTokenInfo.tokenBAmount;
   }, [swapTokenInfo.tokenBAmount, tokenB?.decimals]);
 
+  const hasTokenABalance = useMemo(() => {
+    return swapTokenInfo.tokenABalance !== "-" && swapTokenInfo.tokenABalance !== "0";
+  }, [swapTokenInfo.tokenABalance]);
```

Comment:
In this case, you can also use `include` method.

```suggestion
    const hasTokenABalance = useMemo(
  () => !["-", "0"].includes(swapTokenInfo.tokenABalance),
  [swapTokenInfo.tokenABalance]);
```

Reply:
Currently, we only have 2 comparisons, so I think the existing method is clearer and more efficient. 
But thanks for the suggestion, I think the includes method you suggested will be more maintainable if we have more than 3 values to compare in the future.